conjure-java-core/src/integrationInput/java/com/palantir/product/EteServiceEndpoints.java

```diff
@@ -28,59 +30,227 @@
 import javax.annotation.Generated;
 
 @Generated("com.palantir.conjure.java.services.UndertowServiceHandlerGenerator")
-public final class EteServiceEndpoint implements Endpoint {
+public final class EteServiceEndpoints implements Service {
     private final UndertowEteService delegate;
 
-    private EteServiceEndpoint(UndertowEteService delegate) {
+    private EteServiceEndpoints(UndertowEteService delegate) {
         this.delegate = delegate;
     }
 
-    public static Endpoint of(UndertowEteService delegate) {
-        return new EteServiceEndpoint(delegate);
+    public static Service of(UndertowEteService delegate) {
+        return new EteServiceEndpoints(delegate);
     }
 
     @Override
-    public Routable create(HandlerContext context) {
-        return new EteServiceRoutable(context, delegate);
+    public Registrable create(ServiceContext context) {
+        return new EteServiceRegistrable(context, delegate);
     }
 
-    private static final class EteServiceRoutable implements Routable {
+    private static final class EteServiceRegistrable implements Registrable {
         private final UndertowEteService delegate;
 
         private final SerializerRegistry serializers;
 
-        private EteServiceRoutable(HandlerContext context, UndertowEteService delegate) {
+        private EteServiceRegistrable(ServiceContext context, UndertowEteService delegate) {
             this.serializers = context.serializerRegistry();
             this.delegate = delegate;
         }
 
         @Override
-        public void register(RoutingRegistry routingRegistry) {
-            routingRegistry
-                    .get("/base/string", new StringHandler())
```

Comment:
I like the simplicity on this side, the new values are more difficult for me to read, but I'm not sure that's something we should optimize on.

Reply:
I think `of` was not that bad in terms of conciseness.

Reply:
could provide a `addAll` so that it reads
```
endpointRegistry.addAll(
  Endpoint.get(..),
  Endpoint.post(..));
```